operator/pkg/s3.go

```diff
@@ -1,43 +1,37 @@
 package operator
 
 import (
+	"encoding/json"
 	"fmt"
-	"os"
-
-	"github.com/yetanotherco/aligned_layer/common"
+	"io"
+	"log"
+	"net/http"
 )
 
-func (o *Operator) getBatchFromS3(_batchDataPointer string) ([]VerificationData, error) {
-	// resp, err := http.Get(o.Config.S3BucketConfig.Url + proofId)
-	// if err != nil {
-	// 	return nil, err
-	// }
-	// defer resp.Body.Close()
-
-	// proof, err := io.ReadAll(resp.Body)
-	// if err != nil {
-	// 	return nil, err
-	// }
-
-	// FIXME(marian): We are simulating getting the batch from s3, but this is hardcoded right now
-	proofBytes, err := os.ReadFile("./task_sender/test_examples/gnark_plonk_bn254_script/plonk.proof")
+func (o *Operator) getBatchFromS3(proofUrl string) ([]VerificationData, error) {
+	log.Println("Getting batch from S3..., proofUrl:", proofUrl)
```

Comment:
small nit, but let's use the operator logger here
```suggestion
	o.Logger.Infof(...)
```

Reply:
Changed on 9ca7d24706c933d6234160a67d872e6842f40443